Compile failure in new-updates.

Dear all,

I can't compile the newest updated xtb. Config with meson and gives

```
(base) [xchen@localhost xtb]$ meson setup build_intel --optimization=2
The Meson build system
Version: 0.51.2
Source dir: /home/xchen/Software/xtb-source/xtb
Build dir: /home/xchen/Software/xtb-source/xtb/build_intel
Build type: native build
DEPRECATION: Duplicated values in array option is deprecated. This will become a hard error in the future.
DEPRECATION: Duplicated values in array option is deprecated. This will become a hard error in the future.
DEPRECATION: Duplicated values in array option is deprecated. This will become a hard error in the future.
Project name: xtb
Project version: 6.2.1
Fortran compiler for the host machine: ifort (intel 19.0.5.281 "ifort (IFORT) 19.0.5.281 20190815")
C compiler for the host machine: icc (intel 19.0.5.281 "icc (ICC) 19.0.5.281 20190815")
C++ compiler for the host machine: icpc (intel 19.0.5.281 "icpc (ICC) 19.0.5.281 20190815")
DEPRECATION: Duplicated values in array option is deprecated. This will become a hard error in the future.
Build machine cpu family: x86_64
Build machine cpu: x86_64
Program git found: YES (/usr/bin/git)
Program date found: YES (/usr/bin/date)
Program whoami found: YES (/usr/bin/whoami)
Program hostname found: YES (/usr/bin/hostname)
Configuring xtb_version.fh using configuration
Library pthread found: YES
Library m found: YES
Library dl found: YES
Library mkl_intel_lp64 found: YES
Library mkl_intel_thread found: YES
Library mkl_core found: YES
Library iomp5 found: YES
Library mkl_rt found: YES
Run-time dependency threads found: YES 
Program a2x found: YES (/usr/bin/a2x)
Build targets in project: 5
Found ninja-1.9.0 at /home/xchen/anaconda3/bin/ninja
```


Ninja install the program. ninja -C build_intel test,gives,

```
....

../xtb/model_hessian.f90(1632): remark #15009: model_hessian_mp_strtch2_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(571): remark #15009: model_hessian_mp_mh_swart_outofp_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(1482): remark #15009: model_hessian_mp_outofp2_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(1773): remark #15009: model_hessian_mp_mh_eeq_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(716): remark #15009: model_hessian_mp_mh_lindh_d2_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(852): remark #15009: model_hessian_mp_mh_lindh_stretch_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(955): remark #15009: model_hessian_mp_mh_lindh_bend_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(1168): remark #15009: model_hessian_mp_mh_lindh_torsion_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(1752): remark #15009: model_hessian_mp_fk_lindh_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(1302): remark #15009: model_hessian_mp_mh_lindh_outofp_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(797): remark #15009: model_hessian_mp_mh_lindh_ has been targeted for automatic cpu dispatch
../xtb/model_hessian.f90(2085): remark #15009: ddvopt_ has been targeted for automatic cpu dispatch
[151/260] Compiling Fortran object 'xtb@sta/xtb_copyc6.f.o'.
../xtb/copyc6.f(22): remark #15009: copyc6_ has been targeted for automatic cpu dispatch
ninja: build stopped: subcommand failed.
```

I have no idea about what's going wroing. 

The most confusing thing is that I can compile the old version which in the release page.

--- a/TESTSUITE/cpp_api_example.cpp
+++ b/TESTSUITE/cpp_api_example.cpp
@@ -34,11 +34,11 @@ main (int argc, char **argv)
                                     &energy, nullptr, dipole, q, nullptr, qp, wbo);
 
    assert(stat == 0);
-   assert(fabs(energy + 8.3824793818504) < thr);
-   assert(fabs(q[5] - 0.05184019996829) < thr);
-   assert(fabs(dipole[2] + 0.29832384492435) < thr);
-   assert(fabs(qp[14] - 0.56386138525354) < thr);
-   assert(fabs(wbo[9] - 2.89823984265213) < thr);
+   assert(fabs(energy + 8.3824793818504) < 1.0e-10);
+   assert(fabs(q[5] - 0.05184019996829) < 1.0e-8);
+   assert(fabs(dipole[2] + 0.29832384492435) < 1.0e-6);
+   assert(fabs(qp[14] - 0.56386138525354) < 1.0e-6);
+   assert(fabs(wbo[9] - 2.89823984265213) < 1.0e-8);
 
    return 0;
 }

--- a/meson.build
+++ b/meson.build
@@ -41,6 +41,10 @@ if fc.get_id() == 'intel'
   add_project_arguments('-traceback', language: 'fortran')
 endif
 
+if cxx.get_id() == 'intel'
+  add_project_arguments('-std=c++11', language: 'cpp')
+endif
+
 ## ========================================== ##
 ## LIBRARIES
 ## ========================================== ##

